doc/man7/EVP_SIGNATURE-ML-DSA.pod

```diff
@@ -81,6 +81,16 @@ Note that the message encoding steps from
 L<FIPS 204|https://csrc.nist.gov/pubs/fips/204/final> Algorithm 2 step 10 and
 Algorithm 3 step 5 are omitted when this setting is 1.
 
+=item "mu-only" (B<OSSL_SIGNATURE_PARAM_MU_ONLY>) <integer>
+
+Setting this value to 1 causes EVP_PKEY_sign_message_final() or EVP_PKEY_sign()
+to output the calculated value of "mu" instead of the signature.
+The default value of 0 generates a normal signature.
+EVP_PKEY_verify() ignores this option.
+The returned mu value effectively prehashes the message, and can be supplied as
+input to any cryptographic module that accepts external mu as an input to a
+"ML-DSA" signature operation.
```

Comment:
Suggest adding something along the lines of:

> The return mu value MUST not be used as a message digest.

Reply:
I dont even know what that means,